### PR TITLE
Nerfs legion core healing

### PR DESCRIPTION
**Pull request by anconfuzedrock**
:cl: 
tweak: Legion cores now can only heal 75 brute and 75 burn damage when applied directly to the forehead, which will cause slight brain damage when done repeatedly. implanted cores are as strong as ever.
tweak: lavaland cacti can heal minor brain damage.
/:cl:

[why]: I strongly dislike the idea of mining infinitely better than anything medbay has on offer. Their survival medipens are already better than any pen medbay can get, but cores are absurdly overpowered. This change still allows them extreme healing power inside and outside mining, but they won't fix everything, and spamming them to hell and back would be punished just enough that you can't take down a legion tendril using a toolbox. Using cores sparsely and eating cacti or getting like two mannitol pills from medbay will completely fix this issue. Miners have plenty of heals in their medipens, cacti, and donk pockets. If a miner is concerned because they rely on them to defeat bosses that give them wizard items, they still have a few options:
-carry extra cores if the boss deals massive amounts of burn damage
-get a core surgically implanted, which will still instantly fully heal.
-use the survival medipens that they also have access to
-get chemicals from medbay, either to heal or to increase movement speed to avoid damage slowdown
-bringing help to distract the boss while they heal in a game designed for 50+ players on one server
among others. I think this is for the best.

### CODE REVIEW

**Comment by adamkad1** [comment]
ided plz nerf

**Comment by anconfuzedrock** [comment]
adam is scared that he'll have to say ided plz buff when he can't completely outclass medbay as miner as easily.

**Comment by adamkad1** [comment]
i play doctor more than miner

**Comment by anconfuzedrock** [comment]
well I've never been killed with a legion core, so I guess we're on the same page here.

**Comment by adamkad1** [comment]
well i once did, when i tried to crusher bubblegum with 50%+ time dilation

**Comment by wesoda25** [comment]
Just make them cease to work on the station. If you REALLY want a cool lore reason, how about this: Legion cores draw power from legion. Upon leaving lavaland z level, you get: "The legion core's lights dim as they fall out of range of the great one" etc etc. And, if you beat legion, then legion cores get to work anywhere. Meaning there's finally incentivization to fight the legion, and with whoneedsspace reworking the legion, the fight will be harder than it once was.

**Comment by adamkad1** [comment]
> Just make them cease to work on the station. If you REALLY want a cool lore reason, how about this: Legion cores draw power from legion. Upon leaving lavaland z level, you get: "The legion core's lights dim as they fall out of range of the great one" etc etc. And, if you beat legion, then legion cores get to work anywhere. Meaning there's finally incentivization to fight the legion, and with whoneedsspace reworking the legion, the fight will be harder than it once was.

oh yeah, thats a good idea, there is ABSOLUTELY NO REASON to fight legion

**Comment by anconfuzedrock** [comment]
That's still pretty arbitrary. I'd rather something consistently work everywhere. Legion cores will still be perfectly viable after this change, just not absurdly viable.

**Comment by 81Denton** [comment]
Good PR, it's absurd that you can get admin heals from braindead mobs that respawn en masse. Stabilizing serum is only 400 good boy points too.

**Comment by adamkad1** [comment]
add minor brain damage healing to lavaland cactis or leaves then

**Comment by improvedname** [comment]
Just make a legion core thats expirement not stabilizable like lmfao if you want legion cores to work stabilize in time

**Comment by anconfuzedrock** [comment]
I LITERALLY ADDED BRAIN DAMAGE HEALING TO LAVALAND CACTI. IT'S IN THE CHANGELOG.

maybe I shouldn't have actually, since mannitol isn't very hard to make and you can survive a few cores already.

**Comment by coiax** [comment]
Did you actually change lavaland cacti?

**Comment by anconfuzedrock** [comment]
wait, huh, I didn't commit that. nevermind, sorry.

**Comment by adamkad1** [comment]
> wait, huh, I didn't commit that. nevermind, sorry.

Duuud

**Comment by KomradeSpectre** [comment]
This seems heavily 'IDED', especially when looking at the reasoning.

**Comment by anconfuzedrock** [comment]
Well it isn't ided. I can't prove that, but it can't be proved otherwise, so I'll not worry about it.

**Comment by anconfuzedrock** [comment]
mining may be different from the station, but it's similar enough that 75 hp = a lot of healing. I'd rather not separate mining and the regular station more than they are since the removal of the asteroid, having two different games that only 3 players max can sign up for is just dumb.

**Comment by KomradeSpectre** [comment]
> having two different games that only 3 players max can sign up for is just dumb.

You realize, the public mining dock exists to make it more than just 3 players, right?

**Comment by anconfuzedrock** [comment]
yeah, how often's it used? still a lot of snowflakey stuff for a small fraction of the station. I'd rather it make slightly more sense.

**Comment by adamkad1** [comment]
Yeah, nerfing cores will make remaining miners more robust, and miners are goddamn annoying as is

**Comment by necromanceranne** [comment]
So, exactly what is this PR meant to address? Is the problem with miners possessing a cure-all? Is it related to bringing it onto the station? Or bosses being too easy because of cores?

I don't think the reasoning here is well thought out enough. I'm not necessarily disagreeing that legion cores are abusable to all heck, but I think you need to justify it a bit better.

**Comment by anconfuzedrock** [comment]
I justify it that legion cores are abuseable to all heck, yes.

**Comment by KomradeSpectre** [comment]
A compromise solution: Make it so decayed cores with the stabilizing serum do exactly what you plan here, and leave the non-decayed ones alone.

**Comment by PKPenguin321** [comment]
nerf is warranted, but this seems somewhat extreme. maybe have it heal 50 of each damage type instantly, then another 50 gradually?

**Comment by adamkad1** [comment]
> So, exactly what is this PR meant to address? Is the problem with miners possessing a cure-all? Is it related to bringing it onto the station? Or bosses being too easy because of cores?
> 
> I don't think the reasoning here is well thought out enough. I'm not necessarily disagreeing that legion cores are abusable to all heck, but I think you need to justify it a bit better.

Probally not the third reason. Legion cores are a great save, but they wont magically make you beat the boss if you lack skill. If you get hit, you fucked up or enviroment fucked you up, but you could prepare it better

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> nerf is warranted, but this seems somewhat extreme. maybe have it heal 50 of each damage type instantly, then another 50 gradually?

Or if you're not wanting full heals 25 insta then 50 slow or something.

**Comment by Shadowflame909** [comment]
I'm never going to beat megafauna at this rate, with all of the nerfs and buffs.

**Comment by zeroisthebiggay** [comment]
maybe just make legion cores go inactive when off of the lavaland zlevel or lose most of their power; causing them to heal on a little bit of brute and burn damage when not on the lavaland zlevel

**Comment by Whoneedspacee** [comment]
@Shadowflame909 if you seriously can’t beat the megafauna with how few and far between lavaland updated are then your problem is not the nerfs or buffs.

I have generally a full month before I’ve done other updates (so far at least), so if you somehow haven’t learned to kill any megafauna in that timespan I feel sorry for you.

**Comment by 4dplanner** [comment]
Legion cores on station definitely need nerfing. A miner with stimulum and cores / carp, core, holoparasite/ etcetc is almost unstoppable.

As to on lavaland, this solution has a number of possibly unintended consequences. The 25 burn only is a huge buff to hiero and drake, who mostly deal burn damage, while being not much affecting fighting colossus and bubblegum. It also affects greed ruin, which will become unusable without specific prep.

**Comment by 4dplanner** [comment]
Overall though, ~~fuck you my main~~ probably a good change

**Comment by 4dplanner** [comment]
Also, have you gotten feedback from newer  miners? Cores are a crutch, sure, but I worry that it'll be hard to get into megafauna fighting if every single fuckup in drake arena leads to death unless you spam cores/hiero brutally shreds you, given these are arguably the "starter bosses".

**Comment by Whoneedspacee** [comment]
The 25 burn is so miners are less op against stuff like burn damage. As you said a miner with current cores and stimulum is near unbeatable. With some good damage here you can crit them without spamming being entirely viable if you have a good burn damage weapon. Which entirely are the stations main way of damage to be honest.

Stimulum really should’ve never been buffed to stun immunity anyways. It’s braindead to get and everyone knows how to get it now, fusion is easy peasy. It’s just like old chemistry op shit and needs to be nerfed.

**Comment by AlexTheSysop42** [comment]
1) How are miners getting high-tier gases? (also they aren't easy if you don't know how)
2) Cores are limited by supply, legions are somewhat uncommon unless you find a tendril, and then you need to use the core in 4 minutes or stabilize it. If a traitor miner is abusing cores you just need to wait for their supply to run out and then force them to use any they stabilized.
3) Cores are there only source of instant healing miners have, and very useful for quick de-burning against the ash drake.
4) This significantly buffs ash drake and heiro. (Plus, the latter will already brutally murder you if you get slowdown for longer than a few seconds, making medipens significantly less useful.)

**Comment by SomeguyManperson** [comment]
Why not increase the price of permanent stabilizer to the 2000-4000 range so mining would require more mining for panic heals

**Comment by Whoneedspacee** [comment]
1. It’s really easy to do it, extremely easy now with the new methods.

2. Legion tendrils give a massively supply. Stabilizer serums are not hard to get. 4 minutes is a long time. Inert legion cores can still be stabilized to be useable.

3. This still makes them useable it just makes them less overpowered and gives them weaknesses. It doesn’t heal toxins or heal watcher slowdown or anything like that anymore, and has a drawback. It’s still very strong.

4. Yes I’m glad you can’t just rush 4 legion cores and facetank hierophant. Yes I’m glad you can’t just meme away the drakes fire now.

These items are very overpowered for the status effects they can instantly purge and are spammable.

**Comment by Tlaltecuhtli** [comment]
why 25 burn heal specifically? make it at least reset your body temperature so you can use it after lagging into lava lakes

**Comment by Kromgar** [comment]
How about you just make stabilized cores not do the full heal? Or how about make it so you can't stabilize any cores that have become inert?

Either way i still think the cores should heal more burn damage and put out fires. Legion cores are extremely useful in the situation you lag into a lava lake.

**Comment by 4dplanner** [comment]
Honestly you can get a ton of cores just by running around even without a tendril

**Comment by Whoneedspacee** [comment]
Maybe eventually I’ll implement VR megafauna training so people can stop whining. Eventually.

**Comment by ExcessiveUseOfCobblestone** [comment]
You could just add an argument to full heal so you can pick n choose what legion cores do compared to admin heals.

**Comment by AlexTheSysop42** [comment]
@4dplanner in my experience they're surprisingly rare, hunting for a few minutes (remember, there's a 4 minute timeout) might give 0-2 cores

**Comment by anconfuzedrock** [comment]
Saying I'm nerfing legion cores "into the ground" is absurd. They'll still be incredibly powerful healing items, we had to put a timer on bruise packs and they healed 25 to 1 limb. Legion cores were admin tier powerful for healing before, so yes, reducing them to be only extremely powerful is a large nerf. 
>But if your nerfing the heal so much why the fuck are you gonna have it do brain damage?

Because I don't have to nerf something in specifically one aspect? I have several issues with legion cores, and that fixes the aspect of them being absurdly spammable with no prep, making legion tendrils a complete joke. Let me give an inaccurate analogy. lets talk about holoparasites, which used to cost 12tc and you didn't risk cloneloss damage if you're healed out of crit by one. Those are big nerfs, but holoparasites are still very powerful. It couldn't just be nerfed to cost 18tc, because healing holoparas would still make people far too close to unkillable. I'm not saying this isn't a major nerf to legion cores, but even with this major nerf, they'll still be very powerful. 
>Why 25 burn heal specifically? make it at least reset your body temperature so you can use it after lagging into lava lakes

Watchers drop diamonds because they're supposed to be very dangerous. I don't want burn damage from them to be as easy to ignore. I don't see much reason to reset body temperature. Survival medipens already are designed to help with heat, so that would be a good situation to use a pen instead of the cure-all. You already start with a survival pen anyways, so they aren't too rare to use or anything.

I can see how this buffs ash drakes and the hierophant, but I don't think that's a concern. There are still plenty of ways to deal with them, stopping by medbay for a burn-kit or other meds are still good alternatives that promote actually interacting with the station. An implanted core will still fully heal.

**Comment by 81Denton** [comment]
@ExcessiveUseOfCobblestone revive() has an argument that legion cores use, the difference isn't big though.

https://github.com/tgstation/tgstation/blob/c2148f63a678f7b57a0d9d1e5fd116451d12f18a/code/modules/mob/living/carbon/carbon.dm#L811-L819

**Comment by ExcessiveUseOfCobblestone** [comment]
yeh i have an idea to mess with it a bit that ill look into this evening.

**Comment by Whoneedspacee** [comment]
Getting something like 3-4 legion cores then immediately attacking a boss is not hard at all. Worst case scenario they run out you run away and get more. Stabilizing serums I generally only see used for station use.

**Comment by ATH1909** [comment]
Have you talked to the miner mains about this yet? What about the people who want to get into mining?

**Comment by improvedname** [comment]
When a legion core regrows yoir limbs n shit because it just works but it doesnt heal your depression

**Comment by Shadowflame909** [comment]
Alright gamer. This shall be my last attempt at a swaying argument on this PR. I empathize with what you're trying to accomplish. No one likes a full healing bastard running around tanking hits and tanking damage. No one. They're unfair, Unfun and leave a sour taste in peoples mouths. A part of the reason why lings became the most hated game-mode.


But, put yourself in the shoes of the majority. The shitty miners. We run around on Lavaland trying to avoid instant death with these goliath stuns, the watcher cryo stuns, and the antagonist ash-lizard killing us with spears.

The once great savior of mining, the Legion core, with this PR now only heals 75 brute and 25 burn, whilst slowly killing you with brain damage. Ignoring any extra burn that you may get as you run around in your bulky armored suit that holds in heat/cold. What was great about the Legion Core, was the ability to lose that slow-down.  Ash-storm with a kinetic crusher and have no space for a bluespace beacon? Run with the Legion-Core. Slow down after a couple of times whilst fighting literally any megafauna? Legion-Core!

What I'm trying to say is, Mining is undergoing some heavy-handed changes. Pre-Scanning got made virtually useless (Unless you're trying to steal someone's identity) So Miners no longer have a chance at healing. All the megafauna are undergoing buffs, or have been buffed. (Like the Ash-Drake spawning pools of nigh instant death lava pits.)

All other heals won't save you from the slow-down. With the Legion-Core suffering a drastic nerf, and coming with a downside from this PR. How are we expected to survive? Team-Work? Well, Megafauna's get their own heal from that, after they chomp down on your fellow miner.

That's all I really wanted to say. If this seriously gets merged, I think round-start augments will only be the viable solution as Lava-Land basically becomes doom on the hardest mode. (Pretty sure Augments are going to be hit with a nerf pretty soon as well.) So, all in all. I hope you're looking at the very well suggested alternative of Legion-Cores not working on the Station and being Lava-Land only.

Or as another one, just have them be Lava-Land Only, Anti-Stuns without the KO.  These gradual heals kill the miners.

**Comment by improvedname** [comment]
Hashtag legion

**Comment by MimicFaux** [comment]
Miner healing is rarely 'just spam cores', and for some people they rely on passive medipens to keep them going and the flat healing burst of a core if they hit that dangerous damage-slow.

And no amount of alternative medical supplies are going to help you put your guts back inside your belly when the BDM disembowels you. (A real danger for Crusher users)

**Comment by HydroFloric** [comment]
Hola, miner main here. Just wanted to speak my piece here. Firstly I just wanna say this is a good step forward in making mining less uber OP compared to some other parts of the station, as much as I loved my box of admin heals this change is something that needed to happen. The one thing I would suggest is making it heal a flat 50/50 brute burn but im wont be losing sleep over 75/25.

**Comment by Repukan** [comment]
Just make legions from tendrils not drop cores.

Main justification for the pseudo admin heal was how easy it is to be permanently removed from the round. If you remove the main farming method of cores, then someone will have to spend more time farming them which gives time for the crew to prepare itself (or they can die while farming them this way.)

**Comment by Whoneedspacee** [comment]
@Repukan ‘s idea is good actually. Just make legion cores a very non-renewable resource and maybe up the price of stabilizing serums as well. Then I could live with legion cores being stronger.

**Comment by 4dplanner** [comment]
>stopping by medbay for a burn-kit
The pills are heal over time, and so you're dead, and the patches can't be applied mid combat. I suppose if you stacked cores + medpin + oxandrolone that'd work, though.

**Comment by 4dplanner** [comment]
Still, I think 75/25 is a bad idea. Personally I'd make it heal 50 mixed damage - 50 burn, 50 brute, or 25 of both.

**Comment by 4dplanner** [comment]
As in it adapts to your damage

**Comment by 4dplanner** [comment]
I'm also going to make a pr to restrict cores to lavaland later today

**Comment by HydroFloric** [comment]
Cores not being dropped by tendril legions actually seems like a very good solution to this issue.

**Comment by Shadowflame909** [comment]
What if we just buff legions? The whole gimmick of them is psuedo zombieism

**Comment by 4dplanner** [comment]
Having tendrils not give cores still lets you have multiple instant full heals on station, which just shouldn't be a thing. I think it might work to limit them to lavaland AND prevent tendrilgions from dropping.

**Comment by anconfuzedrock** [comment]
LImiting it to lavaland would be arbitrary and removing cores from tendrils won’t do much, having something widely available and kinda strong is much better than something situational or uncommon and ridiculously overpowered.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
I would prefer the difference between brute/burn to be closer if not even as there's plenty of sources of both in the hellscape.

The brainloss seems derpy. How is that information going to be conveyed to the player (before they have imaginary friends and start getting hypnotized lol)? Does it even need that?

**Comment by anconfuzedrock** [comment]
I  can think of a few ways to convey it, maybe I shoulda in the first place, will do now...

**Comment by Kromgar** [comment]
All i'm saying is no one is bringing up the fact that stabilizing serums can revive inert cores. It never really made much sense to me. This could be an avenue for nerfing the cores. Once they go inert they could just be entirely useless. You could make it so that YOU NEED stabilizing serums immediately when you find one or its likely to become useless

**Comment by HydroFloric** [comment]
> LImiting it to lavaland would be arbitrary and removing cores from tendrils won’t do much

On the contrary, removing cores from tendrils would actually effect a lot as that's where basically all cores are gotten. If you really wanted to as well, decreasing legion spawn rates could also be added.

File: code/modules/mining/equipment/regenerative_core.dm [hunk 1]
```diff
@@ -81,9 +81,11 @@
 				H.visible_message("[user] forces [H] to apply [src]... [H.p_they()] quickly regenerate all injuries!")
 				SSblackbox.record_feedback("nested tally", "hivelord_core", 1, list("[type]", "used", "other"))
 			else
-				to_chat(user, "<span class='notice'>You start to smear [src] on yourself. It feels and smells disgusting, but you feel amazingly refreshed in mere moments.</span>")
+				to_chat(user, "<span class='notice'>You smear [src] on yourself. You still feel disgusting, but your wounds appear to be healed.</span>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mining/equipment/regenerative_core.dm]:
```suggestion
				to_chat(user, "<span class='notice'>You smear [src] on yourself. You still feel disgusting and your head begins to hurt, but your wounds appear to be healed.</span>")
```

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
When you apply it should also hint it does minor brain damage, not just the mood.

**Comment by 4dplanner** [comment]
75/75 will still leave cores as broken on station in the situations they were before.

**Comment by 4dplanner** [comment]
Sure, the brain damage will prevent you spamming literally for ever, but it's still gonna be ridiculous for a while

**Comment by 4dplanner** [comment]
Really, this seems like it'll be a much worse nerf on lavaland than on station, which is a bit backward.

**Comment by Daxxed** [comment]
> 
> 
> Really, this seems like it'll be a much worse nerf on lavaland than on station, which is a bit backward.

I dont think he's playing the same game, or atleast has never used legion cores himself.

**Comment by improvedname** [comment]
add a chem that removes pain slowdown when like a painkiller

**Comment by 81Denton** [comment]
good idea, I'd name it something like "morphine"

**Comment by Kromgar** [comment]
I'd just like to note that this will greatly effect the Greed ruin. As the slot machine causes genetic damage presumably with legion cores healing in mind. So maybe make it burn damage or brute?

**Comment by anconfuzedrock** [comment]
honestly I mighta made the brain damage too low, I had to chew like 10 of them to even get a trauma. I personally think medipens being the gtfo button makes more sense but that's me, I mostly just don't want this to fix things that it doesn't want to.
"this nerfs the greed ruin" is probably the best reason i have heard to make this change. The greed ruin should not be a joke, you should have to actually try if you want to shrug off the clone damage, since the reward can be a free wizard, or miners gibbing themself and not contributing to the station, then proceeding to be an ash walker and whatevs. "This makes it harder for miners to self antag" is a joke to me.

**Comment by 4dplanner** [comment]
>  medipens being the gtfo button 
Rock, I understand where this change is coming from, but how long is it since you last played mining? Heal over time gets you fucked against drake, bubbles, and colossus.

**Comment by anconfuzedrock** [comment]
It's what I'd like to be the case, but not what is. If you're trying to keep runnin though, cores will still work for that.

**Comment by ExcessiveUseOfCobblestone** [comment]
You're the PR author lol

**Comment by Shadowflame909** [comment]
@ExcessiveUseOfCobblestone  Medipens are slow heals, along with every other heal Shaft-Miners can get. The difference is that Medipens don't require you to get fat to use. 

They're versatile and useful after ending Combat, but in combat It's not so great, as it takes a while for it to kick in and help you out.

Legion Cores come in to do the job of a Medipen for you, They help remove that Slowdown so you don't end up dying to the Ash-Drakes Lavapit Arena skill, or Bubblegum's Holoparasite Bubblegum the second, coming at you from 4 different angles.

All in All. The perfect Scenario would be, Medi-Pens as the way to ''Heal'' with Legion Cores as a way to deal with the other things that'll get you screwed over in lavaland, Temperature Changes, Speed-Slowdowns, Broken Limbs.

If the Legion-Core became a Lavaland only Pain-Killer. That's better than morphine as it doesn't KO you. That'd be great. It'd allow the two items to work in sync, and it'd overall be a major improvement, that as far as I can see wouldn't harm anything balance wise.

The current ''Potential Solution'' that baffles me, as it seems more like reasons to avoid Legion-Cores and just round start augment + get meth-pills before mining. I think it'd cause more ''Power-Game'' then intended, as all miners rush anything and everything that could save them from suffering combat slowdowns at the hands of everything that's lethal on lava-land.

Except for the Legion Cores, but with this. Is there any reason to go for the Legions?

Tldr; Temporary Pain-Killer that removes the Combat-Slowdowns, but not the damage. Wouldn't work off Lava-Land, would mesh well with all of the heals in current balance. Wouldn't cause a delay in Mats as Miners all rush Robotics and Chemistry for a new Anti-Stun so they can have a chance at surviving Megafauna and not turning into Gibs.

**Comment by ExcessiveUseOfCobblestone** [comment]
Yeah that sounds like the direction I want to take these two items as I mentioned above.

**Comment by Kromgar** [comment]
@anconfuzedrock I kind of want to self gib myself just to spite you. Then play ashwalker to spite you more.

**Comment by anconfuzedrock** [comment]
ok, go do that then, have fun. I'm sorry if one ruin that's very insignificant ruins this much broader pr for you.

**Comment by 4dplanner** [comment]
Rezadone for greed ruin should work, also gives some interaction with the station rather than singleplayer core farming. It will make greed completion a lot rarer, but it's not like being a wizard is something that miners are owed anyway.

**Comment by AlexTheSysop42** [comment]
Greed ruin is kinda rare and getting wiz from it is 1/20 AFTER getting the die that you can only roll once

**Comment by ExcessiveUseOfCobblestone** [comment]
You're not meant to be guaranteed the greed ruin via legion core spam, you're meant to risk your life for the power (hence the ruin's name).

**Comment by improvedname** [comment]
@Kromgar or just yknow steal a beaker with cryo and buy a luxery pod and a wrench

**Comment by 4dplanner** [comment]
> You're not meant to be guaranteed the greed ruin via legion core spam, you're meant to risk your life for the power (hence the ruin's name).


The  die of fate itself is risky, so it's not like legion cores "guaranteed" a positive outcome. There are other workarounds, so I don't think this PR would totally ruin it, but if healing on the spot isn't used the greed ruin would be pointless. You would be rolling for a tiny chance of getting an item that is then also a gamble - one or the other is fine, which is the current state of affairs, but the consecutive independent risks would be appallingly bad.

**Comment by ExcessiveUseOfCobblestone** [comment]
@4dplanner fine,

You're not meant to be guaranteed the greed ruin's FIRST BARRIER via legion core spam, you're meant to risk your life for the power (hence the ruin's name).

Becoming a fully fledged wizard (some of the other items are good too i guess lol) should be extremely difficult, and being able to cheese 50% of the difficulty guaranteed is lame. Certainly less than 1/20 for simply being a miner. with a hivelord tendril farm.

That said this PR has been up for a week (business hehe) and I'm still not sure what exactly people want out of this item. 

> can someone just describe what exactly they want the cores to be and how they should compare to the medipens?

**Comment by wesoda25** [comment]
Oh boy the PR has hit that point when it’s about to become stale and I can reiterate the points I agree with. 

Legion Cores need a fix, this isn’t it. 

Make legion cores only work on lavaland z level, unless legion is defeated.

Make tendril legions yield no cores. 

OR, make legion cores have a delay to apply, I don’t think anyone mentioned that yet.

**Comment by 4dplanner** [comment]
Legion cores having a delay would kill their use in boss fights as a slowdown remover

**Comment by anconfuzedrock** [comment]
Most of all I don't want cores to be admin heals that fix what they don't need to, and I'd rather medipens and such having less downsides than the core so  that there's actual reason to use the pens over the cores themselves. Thus the it fixing only brute and burn and causing brain damage.

**Comment by improvedname** [comment]
then just make it heal all damage, just not organs/etc internal issues like gutting heart n such or delimbation or even remOVING CUFFS

**Comment by Shadowflame909** [comment]
Or just make it negate damage slow-downs/broken limbs so we don't end up with a useless and self-destructing item.

It would really make this PR something of a synergizer instead of something that's essentially worse than a removal.

**Comment by 4dplanner** [comment]
150 total instant healing is far from useless. I originally was strongly against this PR because 25 burn healing would have created a strange imbalance between bosses, but I think it's pretty reasonable now. I'm not entirely sure about the brain damage part, but I'd have to see it in addition to judge.

**Comment by Shadowflame909** [comment]
Yeah, you're right, I'm being way too temperamental.

I just hope this change doesn't stop Legion Cores from negating temperature changes, otherwise, Ash-Drakes will go from the most killed megafauna, to even more dangerous and more of a rare kill than the Colossus.

Have you seen its upgraded move-set now?

**Comment by 4dplanner** [comment]
I have, but you can still bring a fire extinguisher.

**Comment by Shadowflame909** [comment]
Oh. I guess this PR has no upsides then.

**Comment by ExcessiveUseOfCobblestone** [comment]
This PR isn't stale, i'm just having a hard time deciding if this is the right fix.

**Comment by 4dplanner** [comment]
Can confirm that fire extinguishers have been working fine for me if used while stationary.

**Comment by Shadowflame909** [comment]
Not going to lie, but its starting to seem like Admin-Heals were kind of needed to survive the amount of "Instant Death" these megafaunas can give. Considering it seems like everything else got nerfed to the point where it's your only hope at survival.


Or you can bomb them a whole bunch if you're good at that.

**Comment by Kromgar** [comment]
> I have, but you can still bring a fire extinguisher.

Fire extinguishers require you stand against a wall practically

**Comment by ExcessiveUseOfCobblestone** [comment]
THE PEOPLE HAVE SPOKEN.

Someone else (me or whoever that guy is on discord) will put up a, imo, more proper solution to this.
